github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -45,13 +45,23 @@ presubmits:
           value: "kind-1.19-sriov"
         - name: "KUBEVIRT_NUM_NODES"
           value: "3"
+        - name: "RUN_KUBEVIRT_CONFORMANCE"
+          value: "true"
+        - name: "SONOBUOY_EXTRA_ARGS"
+          value: "--plugin-env kubevirt-conformance.E2E_FOCUS=SRIOV"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - >
-            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM &&
-            make cluster-up
+          nightly_build_base_url="https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt"
+          latest=$(curl -L "${nightly_build_base_url}/latest")
```

Comment:
please add -s to curls

Reply:
Sorry, I forgot to remove this part we already do it from check-cluster-up script.